### ISSUE TITLE
Helix Lava Shortcode "checkboxlist" is missing 2 classes in it's output tags that helix-script.js keys on - which breaks all client-side validation due to exceptions

### ISSUE THREAD

**Issue by stanalyst** [Triumph-Tech/Triumph-Helix#12]
### Description

I have a form inside <lava-form> tags for use with Helix lava application endpoints which implemented several inputs including a Checkbox List (not required) and a Text Box (required - number - min/max/step set). All my inputs are built using the Helix Lava Shortcodes. 

While trying to get client-side validation to work for the Text Box - it would never stop the HX-PUT to the endpoint even when the input for the Text Box was invalid. 

Long story short - discovered that the logic in helix-script.js which validates the Checkbox List inputs (around lines 56-77 of that file) was always throwing an exception ... specifically for the first 2 statements inside the for loop. And the culprits were 2 specific classes that it was looking for: 'cbl-${checkboxListName}' & 'rock-check-box-list' -- when these are missing - exceptions are thrown due to null objects in the chain.

Checking the Lava Shortcode "checkboxlist" - it was not including these 2 classes in the output. I used SQL to set the shortcode "IsSystem" to false - and then edited the shortcode to include these 2 classes - and everything worked. Here is my final template for the shortcode...the changes were made to the classlist of the first <div> tag inside the {[ rockcontrol ]}

```
{% assign sc-values =  value | Split:',',true %}

{[ rockcontrol id:'rc-{{ '' | UniqueIdentifier }}' label:'{{ label }}' showlabel:'{{ showlabel }}' controltype:'{{ controltype }}' isrequired:'{{ isrequired }}' validationmessage:'{{ validationmessage }}' ]}

    <div class="controls js-rockcheckboxlist rock-check-box-list rockcheckboxlist rockcheckboxlist-horizontal in-columns in-columns-{{ columns }} cbl-{{ name }} {% if isrequired %}required{% endif %}">
        {% for item in items %}

            {% assign itemId = id | Append:'_' | Append:forloop.index0 %}
			{% assign isValueSelected = sc-values | Contains:item.value %}
            <label class="checkbox-inline" for="{{ itemId }}">
			    <input id="{{ itemId }}" type="checkbox" name="{{ name }}" value="{{ item.value }}" {% if isValueSelected %}checked{% endif %} {{ additionalattributes }}>
				<span class="label-text">{{ item.text }}</span>
			</label>

        {% endfor %}
    </div>

{[ endrockcontrol ]}
```

### Actual Behavior

Using the Lava Shortcode "checkboxlist" on a Helix form breaks all client-side validation.

### Expected Behavior

Using the Lava Shortcode "checkboxlist" should NOT break client-side validation.

### Steps to Reproduce

- create a Helix form that includes the inputs described above using the shortcodes
- confirm that all client-side validation breaks when using the checkboxlist shortcode as an input

### Issue Confirmation

- [X] Perform a search on the [Github Issues](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ARock) to see if your bug or enhancement is already reported.
- [X] Try to reproduce the problem on a fresh install or on the [demo site](https://rock.rocksolidchurchdemo.com/).

### Rock Version

1.16.6.9

**Comment by stanalyst** [comment]
Forgot another issue I had with this shortcode (and several other ones) ... it was always outputing with a required attribute - even when setting the isrequired parameter to 'false'. Here is my lava template for the shortcode with that fix included ... 

```
{% assign sc-values =  value | Split:',',true %}
{% assign isRequired = isrequired | AsBoolean %}

{[ rockcontrol id:'rc-{{ '' | UniqueIdentifier }}' label:'{{ label }}' showlabel:'{{ showlabel }}' controltype:'{{ controltype }}' isrequired:'{{ isRequired }}' validationmessage:'{{ validationmessage }}' ]}

    <div class="controls js-rockcheckboxlist rock-check-box-list rockcheckboxlist rockcheckboxlist-horizontal in-columns in-columns-{{ columns }} cbl-{{ name }} {% if isRequired %}required{% endif %}">
        {% for item in items %}

            {% assign itemId = id | Append:'_' | Append:forloop.index0 %}
			{% assign isValueSelected = sc-values | Contains:item.value %}
            <label class="checkbox-inline" for="{{ itemId }}">
			    <input id="{{ itemId }}" type="checkbox" name="{{ name }}" value="{{ item.value }}" {% if isValueSelected %}checked{% endif %} {{ additionalattributes }}>
				<span class="label-text">{{ item.text }}</span>
			</label>

        {% endfor %}
    </div>

{[ endrockcontrol ]}
```

I ended up going thru every Helix shortcode and making this same addition/change as needed for each one.

**Comment by Panha-Sim** [comment]
@stanalyst This issue has been fixed in Helix v0.9, which is coming soon.
